rero_ils/modules/documents/views.py

```diff
@@ -597,3 +597,41 @@ def online_holdings(document_pid, viewcode='global'):
         library_holdings.append(record)
         holdings[library['name']] = library_holdings
     return holdings
+
+
+@blueprint.app_template_filter()
+def subjects(subjects, language):
+    """Format the subjects according to the available keys.
+
+    : param subjects: record subjects
```

Comment:
remove white space: `: param` -> `:param`, see https://thomas-cokelaer.info/tutorials/sphinx/docstring_python.html for more details